File: c#/tema_1/ConcertPro/Dao/Model/Spectacle.cs
using System;

namespace ConcertPro.Model
{
    public class Spectacle
    {
        private long SpectacleId { get; set; }
        private string ArtistName { get; set; }
        private DateTime SpectacleDate { get; set; }
        private string SpectaclePlace { get; set; }
        private long SeatsAvailable { get; set; }
        private long SeatsSold { get; set; }

        public Spectacle() { }

        public Spectacle(long spectacleId, string artistName, DateTime spectacleDate, string spectaclePlace, 
            long seatsAvailable, long seatsSold)
        {
            SpectacleId = spectacleId;
            ArtistName = artistName;
            SpectacleDate = spectacleDate;
            SpectaclePlace = spectaclePlace;
            SeatsAvailable = seatsAvailable;
            SeatsSold = seatsSold;
        }
    }
}

File: java/tema_7/backend/src/main/java/ro/mpp/concertpro/model/Spectacle.java
package ro.mpp.concertpro.model;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;

@Entity
@Table(name = "spectacles")
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Spectacle {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "spectacleId")
    private Long spectacleId;

    @Column(name = "artistName")
    private String artistName;

    @Column(name = "spectacleDate")
    private LocalDate spectacleDate;

    @Column(name = "spectaclePlace")
    private String spectaclePlace;

    @Column(name = "seatsAvailable")
    private Long seatsAvailable;

    @Column(name = "seatsSold")
    private Long seatsSold;

}
